.github/workflows/_reusable-aws-setup.yml

```
@@ -0,0 +1,39 @@
+name: Reusable AWS Setup
+description: Centralized AWS credentials and ECR login setup
```

actionlint (1.7.8)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)